app/controllers/users_controller.rb

```diff
@@ -1,4 +1,8 @@
 class UsersController < ApplicationController
   def index
   end
+  
+  def show
+    @user = current_user
```

Comment:
you could also just use current_user in the view to avoid creating another variable